portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,30 +32,38 @@
 // Servlet that returns comments that users input
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  private List<String> comments = new ArrayList<>();
-
-  // Retrieves previously inputted comments
+  // Retrieves previously inputted comments from database
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+
+    PreparedQuery results = datastore.prepare(new Query("Comment"));
```

Comment:
when I make a new obj (eg new Query) should I inline or make a variable to hold?

Reply:
If the query is not needed elsewhere, we usually inline. 
But I think there are some exceptions, e.g. if a long statement like you need something 
datastore.prepare(QueryBuilder.new().setCommends(new Comments()).setXXX).
for long statement case, I usually make a variable for better readability.